.checkov.baseline

```diff
@@ -27,7 +27,8 @@
                 {
                     "resource": "module.extensions-staging.module.alb.aws_security_group.alb-security-group",
                     "check_ids": [
-                        "CKV_AWS_23"
+                        "CKV_AWS_23",
+                        "CKV2_AWS_5"
```

Comment:
Why is `CKV2_AWS_5` still here?

Reply:
Gabe is sorting this one out

Reply:
[DBTP-1039](https://uktrade.atlassian.net/browse/DBTP-1039)